config.json

```diff
@@ -128,6 +128,25 @@
         ],
         "status": "active"
       },
+      {
+        "slug": "rpn-calculator",
+        "name": "RPN Calculator",
+        "difficulty": 4,
+        "uuid": "25cc722b-211d-4271-9381-fdfe16b41301",
+        "concepts": [
+          "vec-stack"
+        ],
+        "prerequisites": [
+          "intro-fn",
+          "numbers",
+          "option",
+          "enums",
+          "match-basics",
+          "loops",
+          "mutability"
+        ],
+        "status": "wip"
```

Comment:
I don't think we need to leave this WIP at this piont; may as well switch it to active now.